src/brain_teasers/python/array_product/README.md

```diff
@@ -0,0 +1,33 @@
+# Array Product Except Self
+
+This function takes an array as input and returns a new array of the same length where each element represents the product of all elements in the original array except the one at the corresponding index.
+
+```
```

Comment:
It will look better if you type python after the backtick, like  ```python

Reply:
I liked it, thank you.
The word 'python' is now added after the backticks.